StacksOfPlates.py

```diff
@@ -0,0 +1,48 @@
+class Stack:
+    def __init__(self, cap):
+        self.stack = []
+        self.cap = cap
+
+    def isFull(self):
+        return len(self.stack) == self.cap
+
+    def push(self, item):
```

Comment:
You could consider returning `true` or `false` here depending on whether the element was added so you don't have to do an additional call to `self.isFull()` in `SetOfStacks.push`